packages/error-reporting/src/google-apis/auth-client.js

```diff
@@ -97,6 +97,17 @@ class RequestHandler extends common.Service {
     }, config);
     this._config = config;
     this._logger = logger;
+
+    var that = this;
+    this.authClient.getToken(function(err, token) {
+      if (err) {
+        that._logger.warn([
```

Comment:
By the time we've gotten here, I think we've determined we will never be able to successfully publish errors. If that is the case, it would be good to log this at the most severe level.